Cargo.toml

```diff
@@ -36,17 +36,16 @@ backtraces = ["cosmwasm-std/backtraces"]
 library = []
 
 [dependencies]
-cw0 = {  version = "0.10.0" }
-cw2 = { version = "0.11" }
-cw20 = { version = "0.10.0" }
-cw20-base = { version = "0.10.0", features = ["library"] }
-cosmwasm-std = { version = "1.0.0-beta" }
-cosmwasm-storage = { version = "1.0.0-beta" }
-cw-storage-plus = "0.10"
-schemars = "0.8.3"
-serde = { version = "1.0.127", default-features = false, features = ["derive"] }
-thiserror = { version = "1.0.26" }
+cw0 = "0.10.3"
+cw2 = "0.15.0"
+cw20 = "0.15.0"
+cosmwasm-schema = "1.1.2"
+cw20-base = { version = "0.15.0", features = ["library"] }
+cosmwasm-std = { version = "1.1.2" }
+cosmwasm-storage = { version = "1.1.2" }
+cw-storage-plus = "0.15.0"
+schemars = "0.8.10"
+serde = { version = "1.0.144", default-features = false, features = ["derive"] }
```

Comment:
```suggestion
```

no longer needed with 1.1. you'll need to update `state.rs` to use the `cw_serde` macro too. :)

Reply:
you are right ff1d54fa0eb5a56ed3127017aba46dd8faa4504e